### ISSUE TITLE
New Logo

### ISSUE THREAD

**Issue by batarian71** [klauscfhq/tusk#51]
Hi, I designed a logo for the media mate application. If you like it, I will give you a present.
https://steemit.com/utopian-io/@batarian71/tusk-new-logo-design

**Comment by jackingpanda** [comment]
I agree, it should have a memorable logo. I could make some of it. Should merge it in this issue, so we don't have to make duplicate issue.

**Comment by Barafu** [comment]
The tray icon in KDE default theme is barely visible. XFCE and Cinnamon use similar colors by default. It should have more contrast on light grey.
